examples/emodel_pipeline_local_python/pipeline.py

```diff
@@ -1,26 +1,66 @@
-"""
```

Comment:
Would the reference included in the license file in the example folder be sufficient for the Python code? 
"The Python code and cell morphology are licensed with the above mentioned CC-BY-NC-SA license."
https://github.com/BlueBrain/BluePyEModel/blob/a503205bcd419fcea8344074b3a53300226ca9b7/examples/emodel_pipeline_local_python/LICENSE.txt#L1C1-L15C4

If this is adequate, I can extend the mention to include .sh and .sbash files as well. Otherwise I can definitely add the license header to all of these files.